.github/workflows/dmd-cxx.yml

```diff
@@ -0,0 +1,99 @@
+# Github action to test C++ implementation of DMD
+#
+# Based-on:
+#  - Github C/C++ with Make
+#  - DMD C++ interop tests
+name: DMD C++
+
+on:
+  pull_request:
+    branches: [ dmd-cxx ]
+  push:
+    branches: [ dmd-cxx ]
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      matrix:
+        # Ubuntu 16.04 in order to support older LLVM / GCC
+        os: [ ubuntu-16.04 ]
+
```

Comment:
Doesn't need to be a matrix row then ?

Reply:
It might be required to add osx and windows later if the auto-tester goes away.  But that is assuming that it will disappear in a matter of months, not years.